src/features/dashboard/components/Modals/AppRegisterSuccessModal/index.tsx

```diff
@@ -39,9 +39,9 @@ export const AppRegisterSuccessModal = ({
       <div className='modal__icon' style={{ background: 'var(--core-color-solid-green-100)' }}>
         <StandaloneCircleCheckRegularIcon fill='#007A22' iconSize='2xl' />
       </div>
-      <div className='modal__text'>
-        <h3 className='modal__header'>Application registered successfully!</h3>
-        <div className='modal__content' style={{ textAlign: 'left', padding: '0px 0px' }}>
+      <div className='modal__content'>
+        <Heading.H4>Application registered successfully!</Heading.H4>
+        <Text>
```

Comment:
This `Text` tag seems like a container which should not be. Please try to use div or alternative if needed.